tools/team_composer.py

```
@@ -0,0 +1,873 @@
+    for c in sorted(characters, key=lambda x: (-x.star, x.name)):
+        star_str = "★" * c.star
+        attach = "/".join(sorted(c.attach_elements)) if c.attach_elements else "-"
+        print(f"  {c.name:<8} {star_str:<8} {c.element:<6} {c.profession:<6} {attach}")
+    print()
+
+    # 搜索最优组合
+    print(f"🔍 搜索最优 {TEAM_SIZE} 人队伍组合 (top-{args.top})...")
+    prefer_elem = args.element
+    prefer_elems = args.element2
+
+    results = find_best_teams(
```

Ruff (0.16.2)

[error] Undefined name `find_best_teams`

(F821)